noir_stdlib/src/embedded_curve_ops.nr

```diff
@@ -67,14 +67,14 @@ impl EmbeddedCurveScalar {
     }
 
     #[field(bn254)]
-    fn from_field(scalar: Field) -> EmbeddedCurveScalar {
+    pub fn from_field(scalar: Field) -> EmbeddedCurveScalar {
         let (a,b) = crate::field::bn254::decompose(scalar);
         EmbeddedCurveScalar { lo: a, hi: b }
     }
 
     //Bytes to scalar: take the first (after the specified offset) 16 bytes of the input as the lo value, and the next 16 bytes as the hi value
     #[field(bn254)]
-    fn from_bytes(bytes: [u8; 64], offset: u32) -> EmbeddedCurveScalar {
+    pub fn from_bytes(bytes: [u8; 64], offset: u32) -> EmbeddedCurveScalar {
```

Comment:
Is this used outside of the stdlib? It's interface doesn't look very user facing.

Reply:
Apparently not. I'll undo this.